src/multi-selector.js

```diff
@@ -55,7 +55,9 @@ angular.module('cp-multi-selector')
 					if(!scope.displayName){
 						function mapName(array) {
 							array.map((item) => {
-								item.display_name = item.name;
+								if (item) {
+									item.display_name = item.name;
```

Comment:
Do we know why item.name is undefined sometimes? Why are we doing that?

Maybe that's something to investigate in the future.